tokens/base.json

```diff
@@ -717,11 +717,11 @@
       "type": "number"
     },
     "400": {
-      "value": "0.65",
+      "value": "0.64",
       "type": "number"
     },
     "500": {
-      "value": "0.85",
+      "value": "0.84",
```

Comment:
Question: Why are these values changing?

Reply:
Originally we were thinking of changing base.unit from `rem` to a unitless "number" token of 4. 

0.85 and 0.65 seemed like arbitrary values when compared against other opacity tokens (0.08, 0.012, 0.40) -- thinking changing the value by 0.01 would make more systems sense. 

But -- open to other thoughts / not changing it if there's a strong reason to avoid